backend/core/Extensions/StringExtensions.cs

```diff
@@ -35,5 +35,10 @@ public static bool IsProduction(this string env)
         {
             return env != null && env.Equals("Production", StringComparison.OrdinalIgnoreCase);
         }
+
+        public static string FormatAsPostal(this string postal)
```

Comment:
Document your functions next time :D

Reply:
Ooh good point, I'll add right now so it will be in the next PR!